newlib/libc/include/stdint.h

```diff
@@ -196,7 +196,7 @@ typedef __uint_least64_t uint_least64_t;
 #define INT32_MIN (-__INT32_MAX__ - 1)
 #define INT32_MAX (__INT32_MAX__)
 #define UINT32_MAX (__UINT32_MAX__)
-#elif defined(__int32_t_defined)
+#elif defined(___int32_t_defined)
```

Comment:
Can you  provide a sample program that demonstrates this problem? I ask because include/sys/_stdint.h defines __int32_t_defined when ___int32_t_defined is set. So, either name *should* work fine.

Reply:
> Can you provide a sample program that demonstrates this problem?

No. I found this by `grep` while working with `___int64_t_defined`

Reply:
I found my issue, it was not related to changes in this PR. You can leave code as it is.

(But such difference in variables names slightly irritates.)